airbyte-integrations/connectors/destination-mssql-v2/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+plugins {
+    id("application")
+    id("airbyte-bulk-connector")
+}
+
+airbyteBulkConnector {
+    core = "load"
+    toolkits = listOf()
+    cdk = "local"
+}
+
+application {
+    mainClass = "io.airbyte.integrations.destination.mssql.v2.MSSQLDestination"
+
+    applicationDefaultJvmArgs = listOf("-XX:+ExitOnOutOfMemoryError", "-XX:MaxRAMPercentage=75.0")
+
+    // Uncomment and replace to run locally
+    //applicationDefaultJvmArgs = listOf("-XX:+ExitOnOutOfMemoryError", "-XX:MaxRAMPercentage=75.0", "--add-opens", "java.base/sun.nio.ch=ALL-UNNAMED", "--add-opens", "java.base/sun.security.action=ALL-UNNAMED", "--add-opens", "java.base/java.lang=ALL-UNNAMED")
+}
+
+val junitVersion = "5.11.3"
+
+configurations.configureEach {
+    // Exclude additional SLF4J providers from all classpaths
+    exclude(mapOf("group" to "org.slf4j", "module" to  "slf4j-reload4j"))
```

Comment:
nonblocking, did we ever find where this dep is coming from? and/or we could add this exclude to the plugin (`airbyte-bulk-connector.gradle`)

Reply:
@edgao I would have to check.  It may not be an issue in this destination (copy pasta here).  If I recall correctly, it may have been one of the hadoop dependencies that we got via Iceberg and maybe even via the CDK because of parquet.